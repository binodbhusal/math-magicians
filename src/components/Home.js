const Home = () => (
  <div className="home-container">
    <h2> Welcome to our Page</h2>
    <p>
      We are here to unlock the world of arithmetic for you and provide a dose of
      math-inspired inspiration. How cool is that? ðŸ˜‰ Brace yourself for some magical
      math moments as we dive into the realm of numbers and equations. Let&rsquo;s ignite the
      spark of curiosity and embark on an extraordinary journey
      of mathematical wonders!âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨.
      <p>
        <h4>A Journey Through Time: The Evolution of Calculators</h4>

        Calculators have come a long way since their inception, revolutionizing
        the way we perform mathematical computations. The history of calculators
        dates back to ancient times when abacuses were used for basic arithmetic operations.
        However, it was not until the 17th century that mechanical calculators,
        such as the Pascaline and the Leibniz Wheel, emerged. The fascinating journey of
        calculators showcases the continuous pursuit of efficiency and convenience
        in mathematical calculations, empowering
        individuals across various fields and shaping the course of technological advancement.
      </p>
    </p>
  </div>
);
export default Home;
