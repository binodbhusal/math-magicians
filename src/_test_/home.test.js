import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Home from '../components/Home';

test('Expect homepage to match working snapshot', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});
