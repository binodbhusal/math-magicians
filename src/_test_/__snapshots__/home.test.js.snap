// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect homepage to match working snapshot 1`] = `
<DocumentFragment>
  <div
    class="home-container"
  >
    <h2>
       WelCome to Our Page
    </h2>
    <p>
      We are here to unlock the world of arithmetic for you and provide a dose of math-inspired inspiration. How cool is that? ðŸ˜‰ Brace yourself for some magical math moments as we dive into the realm of numbers and equations. Letâ€™s ignite the spark of curiosity and embark on an extraordinary journey of mathematical wonders!âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨âœ¨.
    </p>
  </div>
</DocumentFragment>
`;
